resources/testdata/Cargo.toml

```diff
@@ -113,6 +126,10 @@ clap = { version = "2.33", optional = true }
 icu_provider_cldr = { version = "0.1", path = "../../components/provider_cldr", optional = true, features = ["download"] }
 log = { version = "0.4", optional = true }
 simple_logger = { version = "1.11", optional = true }
+globwalk = { version = "0.8", optional = true }
```

Comment:
I know it's `testdata`, but `glob` crate is maintained by the Rust core team and has no dependencies. Could you use it?

Reply:
The README for `globwalk` lists [4 reasons](https://github.com/Gilnaa/globwalk#why-not-glob) why it exists, and 3 of the 4 apply to me:

1. I need `{}` syntax for locales
2. I need to give a list of multiple selectors, gitignore-style
3. https://github.com/rust-lang-nursery/glob/issues/54, a fundamentally important feature request that has been open for almost 5 years
4. Also, I like the API of `globwalk` better

Reply:
Also, not only is this `testdata`, but this is an optional dependency for `testdata` that is downloaded only when `gen-testdata` is run.  So the impact of this dependency is very minimal, to the extent that I didn't consider it when deciding to add it.